plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/babel-expected-output.yml

```diff
@@ -681,6 +681,12 @@ project:
             dependencies:
             - id: "NPM::remove-trailing-separator:1.1.0"
         - id: "NPM::async-each:1.0.6"
+        - id: "NPM::fsevents:1.2.13"
```

Comment:
Commit message typo: "such as for test project*s*"

Reply:
It confuses me to see lines being *added* to existing expected results, whereas the commit message title talks about ignoring stuff... if at all, I would have expected less data, not more.

Reply:
I believe this file does not need any changes.

Reply:
Please see: https://github.com/oss-review-toolkit/ort/pull/10283#discussion_r2079020348

> I believe this file does not need any changes.

The yarn.lock file mentions fsevents:
https://github.com/oss-review-toolkit/ort/blob/507ee30cea999cb6bfd2cdcc2f5bc8e5e4350dd4/plugins/package-managers/node/src/funTest/assets/projects/synthetic/yarn/babel/yarn.lock#L558-L564

IMO the expected result was incorrect before, and is correct now.